tsp_core/src/json_converter/jsonconverter.cpp

```diff
@@ -1,6 +1,68 @@
 #include "jsonconverter.h"
-
+#include "json_converter\taskUtils.h"
+#include <exception>
 JsonConverter::JsonConverter()
 {
 
 }
+TaskResult JsonConverter::fromJson(nlohmann::json j)
```

Comment:
poniżej jest metoda która robi to samo i jest lepsza, niech ta wywołuje tą co robi to samo